babel.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    sourceMaps: true,
+    presets: [
+        [
+            "@babel/preset-env",
+            {
+                targets: {
+                    esmodules: true,
+                },
+                modules: process.env.NODE_ENV === "test" ? "commonjs" : false,
```

Comment:
please could you add a comment to the code to explain what this is doing, and why?